### PR TITLE
Refactored Header - Navbar Component

### PR DESCRIPTION
**Pull request by ogeobubu**
#623

### CODE REVIEW

File: client/src/components/goal interface navbar/NavLayout.js [hunk 1]
```diff
@@ -10,63 +19,65 @@ const GridLayout = styled.div`
...
-  font-style: normal;
-  font-weight: 500;
-  font-size: 12px;
-  line-height: 20px;
-  color: #00b87c;
-  cursor: pointer;
-  margin-right: 1rem;
-  position: relative;
-`;
+// const Sort = styled.button`
```
Comment (abiona01) [inline client/src/components/goal interface navbar/NavLayout.js:22]: Delete the code if you want, is there any reason why you deleted it, then commented it out?

**Review by abiona01 — CHANGES_REQUESTED** [review]

File: client/src/components/goal interface navbar/NavLayout.js (hunk 1 above)
Comment (ogeobubu) [reply client/src/components/goal interface navbar/NavLayout.js:22]: What is currently in the figma design. 
Check the figma design

Comment (abiona01) [reply client/src/components/goal interface navbar/NavLayout.js:22]: If it's to be deleted, delete it. That's all i ask

**Comment by Ose-4g** [comment]
closed cc @abiona01

**Pull request closed without merging**